webapp/components/setting_item_max.jsx

```diff
@@ -71,14 +72,14 @@ export default class SettingItemMax extends React.Component {
 
         var submit = '';
         if (this.props.submit) {
+            const loading = this.props.loading || false;
```

Comment:
Can you rename this property to saving instead of loading so that it's clear what it does?

Also, you can use defaultProps here as well to have it default to false. It's in a static field defined at the bottom of this file instead of in a static function since this file uses the older style.